easybuild/easyconfigs/l/libpsml/libpsml-1.1.7-foss-2016b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpsml'
+version = '1.1.7'
+
+homepage = 'https://launchpad.net/libpsml'
+description = """LibPSML provides a Fortran API to parse files in the
+ PSeudopotential Markup Language (PSML) format."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://launchpad.net/libpsml/trunk/1.1/+download/']
```

Comment:
@pouillon Please use `%(version_major_minor)s` rather than hardcoding `1.1`

Reply:
Done and took note for the future.